packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue

```diff
@@ -9,103 +9,76 @@
       <div v-if="!filteredSpaceMembers.length">
         <h3 class="oc-text-bold oc-text-medium" v-text="$gettext('No members found')" />
       </div>
-      <div
-        v-if="filteredSpaceManagers.length"
-        class="oc-mb-m"
-        data-testid="space-members-role-managers"
-      >
-        <h3 class="oc-text-bold oc-text-medium" v-text="$gettext('Managers')" />
-        <members-role-section :members="filteredSpaceManagers" />
+      <div v-for="(role, i) in availableRoles" :key="i">
```

Comment:
The order of roles seems to be somewhat random. Seems to be stable per space, but from one space to another it's not the same. Any idea how to sort them in a meaningful and stable way?

<img width="443" alt="Screenshot 2024-08-23 at 11 49 23" src="https://github.com/user-attachments/assets/709fec06-b4b4-45a1-907b-c477da2cc9ac">
<img width="442" alt="Screenshot 2024-08-23 at 11 49 31" src="https://github.com/user-attachments/assets/af39d5f4-990e-4c93-9a7b-5f75feb80ed0">

Reply:
Hmm I could at least sort by the amount of permissions a role has. That should then lead to Managers > Editors > Viewers.

Reply:
nice idea!